### PR TITLE
Update index.js

### PR DESCRIPTION
**Pull request by rydje**
Implement extremely important use case

### CODE REVIEW

**Comment by kevinduplant** [comment]
Will conflict with https://github.com/kevinduplant/is-number/pull/1

Please watch carefully the existing PR before submitting important use cases.

File: index.js [hunk 1]
```diff
@@ -10000,5 +10000,6 @@ function isNumber(number) {
   else if(number === 9998 || number === "9998") return true;
   else if(number === 9999 || number === "9999") return true;
   else if(number === 10000 || number === "10000") return true;
+  else if(number === 10001 || number === "10001") return true;
```
Comment (kevinduplant) [inline index.js]: Are we sure this is a number ?

**Review by kevinduplant — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (rydje) [reply index.js]: how can we 100% sure?

Comment (rydje) [reply index.js]: let me make it more clear

Comment (kevinduplant) [reply index.js]: Since we are not sure. And due to the risk of this PR. We decided to close it. Please come back with a better solution

**Pull request closed without merging**